web/src/MainDemo.elm

```diff
@@ -19,15 +19,17 @@ initCmds : Cmd Msg
 initCmds =
     Cmd.batch <|
         [ perform SetTime Time.now
+        , perform LocalStorageRead (succeed lSKeys.debugLog)
         ]
 
 
 subscriptions : Model -> Sub Msg
 subscriptions model =
     Sub.batch
-        [ Ports.receipt Receipt
-        , Ports.confirmation Confirm
-        , Ports.receive Receive
+        [ Ports.receipt BlockchainReceipt
+        , Ports.confirmation BlockchainConfirm
+        , Ports.receive BlockchainReceive
+        , Ports.gotLocalStorageImpl LocalStorageReceive
```

Comment:
Do these need to be decoded? They will take a (String -> Msg) or (Value -> Msg) but for stuff like the error log we probably want to pass in a decoding function (i.e. a (Value -> Msg))